getting-started.md

```diff
@@ -0,0 +1,90 @@
+This guide provides the first steps to using Anbox Cloud. If you haven't installed Anbox Cloud, please refer to the [installation page](https://discourse.ubuntu.com/t/installation-quickstart/17744) first.
+
+## Access to AMS
+
+For all subsequent commands using the `amc` tool to work you need to access the `ams/0` machine on a regular Anbox Cloud deployment. You can do this by opening an SSH session with the `juju` command:
+
+```bash
+$ juju ssh ams/0
+```
+
+If you're running the Anbox Cloud Appliance instead, you can find the `amc` tool directly on the host and it's already set up to talk to the deployed AMS service.
+
+Alternatively you can also install the `amc` command on your local Ubuntu-based development machine. See [AMS Access](https://discourse.ubuntu.com/t/managing-ams-access/17774) for more details.
+
+## Ensure Images are Available
+
+As a next step you can check that **AMS** has synchronised all images from the Canonical hosted image server. You can list all synchronised images with the `amc image ls` command:
+
+```bash
++----------------------+-----------------------------+--------+----------+--------------+---------+
+| ID                   | NAME                        | STATUS | VERSIONS | ARCHITECTURE | DEFAULT |
++----------------------+-----------------------------+--------+----------+--------------+---------+
+| c4b4djkrorjohh948dfg | bionic:android11:arm64      | active | 1        | aarch64      | true    |
```

Comment:
I updated this table to be more recent, but noticed afterwards I see only the Arm images. Does this make sense, or should the table show the amd ones?

Reply:
Showing one of both is fine. We could even mix arm and x86 if we want as all these variants are possible.